docs/2-deployments/7-multitenant-deployments.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Multi-Tenant Deployments
+---
+
+# Multi-Tenant Deployments
+
+Multi-tenant deployments are when
```

Comment:
are when...?

Reply:
😱  oh my god that's embarrassing- thanks for catching this 🤦